### PR TITLE
Fixes #21223: Move create node doc in rudder core

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/21223

### CODE REVIEW

File: src/reference/modules/usage/pages/advanced_node_management.adoc [hunk 1]
```diff
@@ -1,5 +1,66 @@
...
+It's typically what you are looking for in your provisioning and integration patterns.
+
+A node need at least the following information:
+
+- UUID,
+- hostname,
+- status,
+- OS: type, name
+
+Other information are possible. See the https://docs.rudder.io/api/#tag/Create-Node[plugin API documentation] for more information.
```
Comment (ElaadF) [inline src/reference/modules/usage/pages/advanced_node_management.adoc:18]: change the URL for the API

File: src/reference/modules/usage/pages/advanced_node_management.adoc [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+
+Whatever the final status, a node is created in pending status. Then, if the status is "accepted", node acceptation is automatically and immediately triggered. This ensure that nodes added by that API also follow the standard Rudder validation rules and dynamic group/policies are correctly updated.
+
+=== Request format
+
+The plugin provides a new API endpoint: PUT `/api/latest/nodes`
```
Comment (ElaadF) [inline src/reference/modules/usage/pages/advanced_node_management.adoc:24]: Modify if we decide to use another URL

File: src/reference/modules/usage/pages/advanced_node_management.adoc [hunk 3]
```diff
@@ -1,5 +1,66 @@
...
+
+So for example, for curl: `curl -k -H "X-API-Token: 5YJnb4EenqDHzwx0U0eTPWWhgjTJ8mkk" -H "Content-Type: application/json" -X PUT 'https://your-server/rudder/api/latest/nodes' -d@/path/to/node.json`
```
Comment (ElaadF) [inline src/reference/modules/usage/pages/advanced_node_management.adoc:26]: here too

**Comment by ElaadF** [comment]
PR updated with a new commit

**Review by fanf — APPROVED** [review]

**Pull request merged**